drivers/hmis/app/models/hmis/form/definition.rb

```diff
@@ -473,53 +496,47 @@ def introspect_custom_data_element_definitions(set_definition_identifier: false)
     cdeds_by_key = cded_scope.index_by(&:key)
 
     cded_records = []
-    recur_traverse = lambda do |items|
```

Comment:
This change isn't strictly needed but seemed better to make use of walk_definition_nodes rather than a lambda.